apps/gallery/style/info.css

```diff
@@ -40,6 +40,14 @@
   border-bottom: none;
 }
 
+.no-border {
+ border-bottom: none !important;
+}
+
+.hide {
+  display: none !important;
+}
+
```

Comment:
Thanks for suggesting, this is definitely better. Keeping no-border class as 
label.previousElementSibling.classList.remove('no-border') is cleaner than inline style
label.previousElementSibling.style.borderBottom = '0.1rem solid rgba(255,255,255,0.1)'